src/ring_theory/adjoin_root.lean

```diff
@@ -121,7 +121,8 @@ lemma coe_injective : function.injective (coe : K → adjoin_root f) :=
 variable (f)
 
 lemma mul_div_root_cancel :
-  (X - C (root f)) * (f.map (of f) / (X - C (root f))) = f.map (of f) :=
+  ((X - C (root f)) * (f.map (of f) / (X - C (root f))) : polynomial (adjoin_root f)) =
+    f.map (of f) :=
```

Comment:
This was now timing out, so I added a type annotation which makes is instant again...?